pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -141,11 +141,16 @@ func doUpdate(ctx *zedmanagerContext,
 			snappedAppInstanceConfig, err := triggerRollback(ctx, status)
 			if err != nil {
 				errDesc := types.ErrorDescription{}
+				errDesc.ErrorTime = time.Now()
 				errStr := fmt.Sprintf("doUpdate(%s) triggerRollback failed: %s", uuidStr, err)
 				errDesc.Error = errStr
 				log.Error(errStr)
+				status.SnapStatus.HasRollbackRequest = false
+				setSnapshotStatusError(status, status.SnapStatus.ActiveSnapshot, errDesc)
+				errDesc.ErrorSeverity = types.ErrorSeverityWarning
```

Comment:
Should not be that assignment of errDesc done before the actual call of the setSnapshotStatusError?